index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <title>My First Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a class="welcome-message">Welcome</a>
+            <div>
```

Comment:
- [ ] Since this link acts, as the logo in your web page, it should be inside the header but outside the `nav` element. Kindly place it before the `nav` element.